content/docs/command-reference/data/status.md

```diff
@@ -0,0 +1,133 @@
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+```
+
+As shown above, the `dvc data status` displays changes in multiple categories:
+
+- _Not in cache_ indicates that the hash for files are recorded in `dvc.lock`
+  and `.dvc` files but the corresponding cache files are missing.
+- _DVC committed changes_ indicates that there are changes that are committed
+  that differs with the last Git commit. There might be more detailed state on
+  how each of those files changed: _added_, _modified_, and _deleted_.
+- _DVC uncommitted changes_ indicates that there are changes in the working
+  directory that are not `dvc commit`-ed yet. Same as _DVC committed changes_,
+  there might be more detailed state on how each of those files changed.
+- _Untracked files_ shows the files that are not being tracked by DVC and Git.
+  This is disabled by default, unless [`--untracked-files`](#--untracked-files)
+  is specified.
+- _DVC Unchanged files_ shows the files that are not changed. This is not shown
+  by default, unless [`--unchanged`](#--unchanged) is specified.
+
+By default, `dvc data status` shows changes to tracked files and directories.
+With [`--granular`](#--granular), it will show changes in individual files
+inside those directories.
+
+## Options
+
+- `--granular` - show granular, file-level information of the changes for
+  DVC-tracked directories. By default, DVC only shows the changes at the level
+  where the data is being tracked at.
+
+- `--untracked-files` - show files that are not being tracked by DVC and Git.
+
+- `--unchanged` - show unchanged DVC-tracked files.
+
+- `--json` - prints the command's output in easily parsable JSON format, instead
+  of a human-readable output.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+```dvc
+$ dvc data status
+Not in cache:
+  (use "dvc pull <file>..." to download files)
+        data/data.xml
+
+DVC committed changes:
+  (git commit the corresponding dvc files to update the repo)
+        modified: data/features/
+
+DVC uncommitted changes:
+  (use "dvc commit <file>..." to track changes)
+        deleted: model.pkl
+(there are other changes not tracked by dvc, use "git status" to see)
+```
+
+This shows that the `data/data.xml` is missing from the cache, `data/features/`
+a directory, has changes that are being tracked by DVC but is not git-committed
+yet, and a file `model.pkl` has been deleted from the workspace. The
+`data/features/` directory is modified, but there is no further details to what
+changed inside. The `--granular` option can provide more information on that.
```

Comment:
IMO we can drop this example since it's already shown above, and I don't thin further explanation is needed.

Reply:
Let's keep it if you don't have strong opinion. The granular output follows from this example, it's easier if they are together.